x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/step_define_rule/translations.tsx

```diff
@@ -234,6 +234,21 @@ export const EQL_SEQUENCE_SUPPRESSION_GROUPBY_VALIDATION_TEXT = i18n.translate(
   }
 );
 
+export const MACHINE_LEARNING_SUPPRESSION_DISABLED_LABEL = i18n.translate(
+  'xpack.securitySolution.detectionEngine.createRule.stepDefineRule.machineLearningSuppressionDisabledLabel',
+  {
+    defaultMessage: 'Machine Learning jobs must be running to enable alert suppression.',
```

Comment:
While this is technically true, I think we can still be more direct and tell users that they must act to enable suppression. 

```suggestion
    defaultMessage: 'To enable alert suppression, start one or more Machine Learning jobs.',
```

Another, more general option: 
To enable alert suppression, start Machine Learning jobs.